packages/components/src/DateTimePickers/pickers/DateTimePicker/DateTimePicker.scss

```diff
@@ -7,3 +9,33 @@
 		outline: 1px solid black;
 	}
 }
+
+.footer {
+	:global {
+		.btn-link {
+			color: $st-tropaz;
+			text-decoration: none;
+
+			:hover {
+				background-color: rgba($st-tropaz, 0.1);
+				transition: opacity 0.1s ease-in;
+			}
+			:focus {
+				background-color: rgba($st-tropaz, 0.2);
+			}
+		}
+	}
+}
+
+.btn-today {
+	color: $st-tropaz;
+	font-weight: bold;
+	text-transform: uppercase;
+	padding: 0;
+	height: 25px;
+	> span {
+		border-radius: 4px;
```

Comment:
there is a variable theme for that I guess, look for 'border-radius'

Reply:
thanks, updated already. :-)

Reply:
css is moved to tertiary button, theme package